app/views/ExposureHistory/__snapshots__/Calendar.spec.tsx.snap

```diff
@@ -942,7 +941,8 @@ exports[`Calendar renders 1`] = `
             style={
               Object {
                 "alignItems": "center",
-                "borderColor": "rgba(0, 0, 0, 0)",
```

Comment:
Probably not related to your pr, but something is wonky with this snapshot

Reply:
indeed; there's something non-deterministic going on 🤔

Reply:
Perhaps there's something on this screen that renders a certain way depending on the current date/time?